ports/supergaelicdodgeball/port.json

```
@@ -0,0 +1,30 @@
+{
+    "version": 1,
+    "name": "supergaelicdodgeball.zip",
+    "items": [
+        "Super Gaelic Dodgeball.sh",
+        "supergaelicdodgeball"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Super Gaelic Dodgeball",
+        "porter": [
+            "Zehmaluco"
+        ],
+        "desc": "Mildly violent Dodgeball game made for Awful Winter Jam 2018",
+        "desc_md": null,
+        "inst": "Get the game on shoehead.itch.io/sgdb and place all game files on the gamedata inside supergaelicdodgeball folder",
+        "inst_md": null,
```

Probably a good idea to add "inst_md" with an actual link to the game:
```suggestion
        "inst_md": "Get the game on the [shoehead itch.io page](https://shoehead.itch.io/sgdb), and place all game files in the ports/supergaelicdodgeball/gamedata/ directory.",
```